yogstation/code/modules/jobs/job_types/_job.dm

```diff
@@ -17,13 +17,13 @@
 			var/type = donor_start_items[C.prefs.donor_hat]
 			var/obj/hat = new type()
 			hat.forceMove(BP)
-			
+
 	if(C.prefs.donor_item)
 		var/obj/item/storage/backpack/BP = locate(/obj/item/storage/backpack) in H.GetAllContents()
 		if(BP)
 			var/type = donor_start_tools[C.prefs.donor_item]
 			var/obj/item = new type()
-			item.forceMove(BP)
+			item.forceMove(put_in_hands)
```

Comment:
Nick

Do you even

do you even know how to code

what

my god damn it

fucking

```suggestion
			H.put_in_hands(item)
```

Reply:
REEEEEEEEEEEEEEEEEEEEEEEEEEE